src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
+for (let i = 0; i < li.length; i++) {
+  const span = document.createElement('span');
+  const firstChild = li[i].firstChild;
+
+  if (firstChild.textContent !== undefined) {
+    span.textContent = firstChild.textContent;
+    li[i].insertBefore(span, firstChild);
+    firstChild.textContent = '';
+  }
+}
+
+const spans = document.querySelectorAll('span');
+
+for (let i = 0; i < li.length; i++) {
```

Comment:
Your code works, but it's a little bit complicated, you don't need to iterate all over spans in HTML. This code can be simplified by checking the listener on whole tree and check if it not span

```suggestion
tree.addEventListener('click', (e) => {
  if (e.target.tagName !== 'SPAN') {
    return;
  };

  const item = e.target.nextSibling;

  item.style.display === 'none'
    ? item.style.display = 'block'
    : item.style.display = 'none';
});
```